frontend/app/src/people/widgetViews/organization/RolesModal.tsx

```diff
@@ -39,29 +59,59 @@ const RolesModal = (props: UserRolesModalProps) => {
         borderRadius: '50%'
       }}
     >
-      <Wrap newDesign={true}>
-        <ModalTitle>Add user roles</ModalTitle>
+      <UserRolesWrap newDesign={true}>
+        <UserImage
+          style={{
+            height: '80px',
+            width: '80px',
+            marginLeft: 'auto',
+            position: 'fixed',
+            left: '50%',
+            top: `55px`,
+            transform: 'translate(-40px, -40px)',
+            borderStyle: 'solid',
+            borderRadius: '50%',
+            borderWidth: '4px',
+            borderColor: 'white'
+          }}
+          src={user?.img || avatarIcon}
+        />
+        <UserRolesName>{user?.unique_name}</UserRolesName>
+        <div
+          style={{
+            backgroundColor: '#EBEDEF',
+            height: '1px',
+            width: '100%',
+            margin: '5px 0px 20px'
+          }}
```

Comment:
same with this we should make styled component

Reply:
Sounds good. I'm just getting back after a couple days off, so I'll hopefully have a fix for everything by this afternoon. I was having trouble with positioning the profile picture using position: 'fixed'. Do you know if there's another way to do this relative to the Modal div?